Made a responsive version of Guests reviews block.

marianichiporuk opened the pull request:
Made price info in this block as animation while hovering. Changed yellow underline below titles (:after element): it appears in every section while user scrolls through the site.

marianichiporuk committed:
Made a responsive version of Guests reviews block.
Made price info in this block as animation while hovering.
Changed yellow underline below titles (:after element): it appears in every section while user scrolls through the site.

almuly commented on styles/styles.css:
```diff
@@ -1072,6 +1257,90 @@ body {
+    display: none;
+  }
+
+  .reviews__kingkong {
```
Don't use names of images in names of classes, you can use abstract name and pseudoclasses (:last-child, nth-child, etc)

almuly commented on styles/styles.css:
```diff
@@ -1072,6 +1257,90 @@ body {
+    display: none;
+  }
+
+  .reviews__rokoko {
```
The same, and you not need in two classes with the same properties in one block

marianichiporuk committed:
Deleted some class names for css styling and used :nth-child instead.
Added focus behavior in search form.
Replaced some links into buttons.
Changed the structure of code in cards with countries and places. Fixed animation in cards in every block (now you can hover at any space of whole card and animation will appear).

marianichiporuk committed:
Deleted index files that are not connected to the current project.
